bitlbee.h

```diff
@@ -103,6 +103,13 @@ extern "C" {
 #define g_strcasecmp g_ascii_strcasecmp
 #define g_strncasecmp g_ascii_strncasecmp
 
+/* g_memdup() deprecated as of glib 2.68.0 */
+#ifdef GLIB_VERSION_2_68
```

Comment:
rather than defining a custom macro, could we just use g_memdup2 and define that as g_memdup on anything where glib < 2.68?

Reply:
Yeah, that's fine.  Didn't know what the favored style was in this project or how far back you wanted to support in glib.  I will update the PR.

Reply:
Please also rebase - we've now got github actions set up.

Reply:
Done.